src/components/NavigationGroup.svelte

```diff
@@ -1,6 +1,6 @@
 {{#if separator == 'top'}}<hr />{{/if}}
 {{#if group.length}}
-<ul class='coz-nav-group'>
+<ul class='{{`coz-nav-group${wrappingClass}`}}'>
```

Comment:
I don't get this, maybe it's just the syntax but won't it write `coz-nav-group` if not wrapping and `coz-nav-group--wrapping` if wrapping ? If so, you'll lose the style for `.coz-nav-group` and why in the CSS it is written just`.--wrapping`? I'm confused. 😕